_data-prepper/pipelines/configuration/processors/json.md

```diff
@@ -8,13 +8,15 @@ nav_order: 45
 
 # json
 
+<!--- Should we add "Parse" to the title? The original is "Parse JSON processor. --->
```

Comment:
@graytaylor0 Should we add "Parse" to the tile? The original is "Parse JSON processor".

Reply:
I think it makes sense to have this name be whatever name is expected in the config (in this case it is named `parse_json` so we should change it to that exactly)